### ISSUE TITLE
InternalError (see above for traceback): Blas SGEMM launch failed : m=43264, n=32, k=64 	 [[Node: conv2d_3/convolution = Conv2D[T=DT_FLOAT, _class=["loc:@batch_normalization_3/cond/batchnorm/mul_1/Switch"], data_format="NHWC", dilations=[1, 1, 1, 1], padding="SAME", strides=[1, 1, 1, 1], use_cudnn_on_gpu=true, _device="/job:localhost/replica:0/task:0/device:GPU:0"](leaky_re_lu_2/LeakyRelu/Maximum, conv2d_3/kernel/read)]]

### ISSUE THREAD

**Issue by li829** [qqwweee/keras-yolo3#477]

**Comment by Angel0003** [comment]
在运行yolo_video.py的时候，遇到和你同样的问题，请问你解决了吗？

**Comment by Angel0003** [comment]
When running yolo_video.py, I encountered the same problem as you. Did you solve it?

**Comment by li829** [comment]
这个问题好久了，我忘了具体是哪里。但是，主要就是两种情况吧，一种是导入的权重文件和模型定义的输入图片大小不同；另一种是运行时定义的图片大小和导入的权重文件不一样。对了，也有可能是cuda或者TensorFlow版本的问题，因为当时实验室来了台比较新的机器显卡2080ti所以cuda和tensorflow的版本要高一点，分别是cuda9.2以上和tensorflow1.8以上。

**Comment by Angel0003** [comment]
嗯嗯，我的也是RTX 2080Ti，我的cuda是9.0，cudnn是7.0。5,是不是版本太低了？

**Comment by Angel0003** [comment]
我的tf是1.8.0,keras是2.2.4

**Comment by li829** [comment]
cuda直接上10吧，或者9.2，如果使用anaconda的话直接升级cudatoolkit就行了

**Comment by Angel0003** [comment]
能加个qq吗973616491

**Comment by Angel0003** [comment]
版本需要这么高吗，p5000安装cuda8都没问题

**Comment by li829** [comment]
可能是2080ti的架构不太一样吧，cuda8到10我都试过了，只有9.2以上才行

**Comment by Angel0003** [comment]
你现在用的是9.2的cuda吗，你cudnn和tf都是什么版本，在现在的环境下你能跑通这个代码吗

**Comment by Angel0003** [comment]
你现在用的是9.2的cuda吗，你cudnn和tf都是什么版本，在现在的环境下你能跑通这个代码吗

**Comment by li829** [comment]
对, cuda9.2，cudnn是anaconda自动配的貌似是7.3吧，tf1.8能跑通

**Comment by Angel0003** [comment]
你不是说你试的cuda10不行吗，是不是安9.2就行。cudnn版本不是自己去官网下载安装吗，anaconda能自动配吗

**Comment by li829** [comment]
cuda10是可以的。我用的是linux的anaconda 所以会自动配好。装tf的时候应该是默认装9.0但是你需要收到升级到10或者9.2

**Comment by hyhouyong** [comment]
> 你现在用的是9.2的cuda吗，你cudnn和tf都是什么版本，在现在的环境下你能跑通这个代码吗
ubuntu18.04+GPU2080ti+cuda10.0+cudnn7.6+tensorflow-gpu1.13.1+anaconda3.4+python3.5

**Comment by li829** [comment]
我觉得你的环境应该可以跑通的，cuda9.2和10我都用过

**Comment by hyhouyong** [comment]
> 我觉得你的环境应该可以跑通的，cuda9.2和10我都用过

您好，现在我的服务器有4个卡，均为2080ti，训练时只有一个卡工作，对于多GPU训练，有指定的解决方案吗？

**Comment by Alfred-Credo** [comment]
> cuda10是可以的。我用的是linux的anaconda 所以会自动配好。装tf的时候应该是默认装9.0但是你需要收到升级到10或者9.2

您好，请问怎样用anaconda升级cudatoolkit呢？同时需不需要处理cudnn呢？
我电脑安装的是cuda10.0，但是conda环境里是cudatoolkit9.0

**Comment by li829** [comment]
conda upgrade cudatoolkit
或者
conda update cudatoolkit
或者
conda install cudatoolkit=10.0

**Comment by Angel0003** [comment]
这个问题最终解决了，是tensorflow版本和cuda不匹配造成的，需要装低一些的tf版本，不知道装哪个版本的tf就挨个去试。如果换tf还是不行，也有可能是cuda没装好。楼主通过换tf版本和重装cuda解决的。如果cuda没问题，最好不要轻易动cuda，换个tf版本就可以解决。

**Comment by 2042644107** [comment]
@Angel0003 您好，您最后的配置是什么呀？我遇到同样问题，好多天了，现在都快没头绪了，麻烦您指点一下，谢谢

**Comment by MaverickMeerkat** [comment]
As angel0003 (and google translate) said: I downgraded my tensorflow to 1.6, and the problem was solved. It was 1.10 before.

**Comment by 2042644107** [comment]
thanks for your reply.




------------------&nbsp;原始邮件&nbsp;------------------
发件人:&nbsp;"David Refaeli"<notifications@github.com&gt;;
发送时间:&nbsp;2019年10月27日(星期天) 凌晨0:11
收件人:&nbsp;"qqwweee/keras-yolo3"<keras-yolo3@noreply.github.com&gt;;
抄送:&nbsp;"💕Leo、Pisces"<2042644107@qq.com&gt;;"Comment"<comment@noreply.github.com&gt;;
主题:&nbsp;Re: [qqwweee/keras-yolo3] InternalError (see above for traceback): Blas SGEMM launch failed : m=43264, n=32, k=64   [[Node: conv2d_3/convolution = Conv2D[T=DT_FLOAT, _class=["loc:@batch_normalization_3/cond/batchnorm/mul_1/Switch"], data_format="NHWC", dilations=[1, 1, 1, 1], padding="SAME", strides=[1, 1, 1, 1], use_cudnn_on_gpu=true, _device="/job:localhost/replica:0/task:0/device:GPU:0"](leaky_re_lu_2/LeakyRelu/Maximum, conv2d_3/kernel/read)]] (#477)




As angel0003 (and google translate) said: I downgraded my tensorflow to 1.6, and the problem was solved. It was 1.10 before.

**Comment by lw17809213284** [comment]
> 这个问题最终解决了，是tensorflow版本和cuda不匹配造成的，需要装低一些的tf版本，不知道装其中版本的tf就挨个去试。如果换tf还是不行，也有可能是cuda没装好。如果cuda没问题，最好不要轻易动cuda，换个tf版本就可以解决。

你好，请问您的tensorflow和cuda版本是什么

**Comment by 2042644107** [comment]
感谢您的回复，问题解决了，是tensorfow的问题，cuda10.0不能用pip安装1.12.0,需要用一位大佬在github上发布的1.12.0




------------------&nbsp;原始邮件&nbsp;------------------
发件人:&nbsp;"lw17809213284"<notifications@github.com&gt;;
发送时间:&nbsp;2019年10月30日(星期三) 中午12:15
收件人:&nbsp;"qqwweee/keras-yolo3"<keras-yolo3@noreply.github.com&gt;;
抄送:&nbsp;"💕Leo、Pisces"<2042644107@qq.com&gt;;"Comment"<comment@noreply.github.com&gt;;
主题:&nbsp;Re: [qqwweee/keras-yolo3] InternalError (see above for traceback): Blas SGEMM launch failed : m=43264, n=32, k=64   [[Node: conv2d_3/convolution = Conv2D[T=DT_FLOAT, _class=["loc:@batch_normalization_3/cond/batchnorm/mul_1/Switch"], data_format="NHWC", dilations=[1, 1, 1, 1], padding="SAME", strides=[1, 1, 1, 1], use_cudnn_on_gpu=true, _device="/job:localhost/replica:0/task:0/device:GPU:0"](leaky_re_lu_2/LeakyRelu/Maximum, conv2d_3/kernel/read)]] (#477)



 
这个问题最终解决了，是tensorflow版本和cuda不匹配造成的，需要装低一些的tf版本，不知道装其中版本的tf就挨个去试。如果换tf还是不行，也有可能是cuda没装好。如果cuda没问题，最好不要轻易动cuda，换个tf版本就可以解决。
  
你好，请问您的tensorflow和cuda版本是什么

**Comment by goodapple8946** [comment]
这个问题也卡了我半天，结果把CUDA9.0官网那4个补丁安装就好了。。。
